config/test_dtm_config.yaml

```diff
@@ -1,3 +1,5 @@
 clients: 1
 samples: 500
 size: '128Mb'
+m0d_pids: ['0x4', '0x7']
```

Comment:
@ApurwaMohite  We should not be hard coding these pids, as they may change based on number of cvg.